### PR TITLE
JP : oopsies and fixes

### PR DESCRIPTION
**Pull request by BellezaEmporium**
- Fix issues linked to mov3 services by updating server links
- Fix an issue linked to an old IP being used on BS-TBS on hlsok
- Remove 2 indonesian server links, as one was AniPlus
- Specify that Animax Japan [Indonesian subs] has -1 year of difference in terms of programmation (see .ts files when connecting)

### CODE REVIEW

**Review by Central302 — DISMISSED** [review]
BS-TBS is Work. But mov3 Links Buffering Server Side.

**Comment by LaneSh4d0w** [comment]
Yup, not something I can change whatsoever, maybe some streaming issues on mov3's side.

**Review by Central302 — APPROVED** [review]

File: streams/jp.m3u [hunk 1]
```diff
@@ -46,14 +42,14 @@ http://51.79.193.108:8080/hlsok/j00050/index.m3u8?type=jp
...
+#EXTINF:-1 tvg-id="FujiTV.jp" status="",Fuji TV (540p) [Not 24/7]
+https://fujitv2.mov3.co/hls/fujitv.m3u8
 #EXTINF:-1 tvg-id="GAORA.jp" status="online",GAORA (540p) [Not 24/7]
 http://51.79.193.108:8080/hls/j00045/index.m3u8?type=jp
 #EXTINF:-1 tvg-id="GAORA.jp" status="online",GAORA (540p) [Not 24/7]
 http://51.79.193.108:8080/hlsok/j00045/index.m3u8?type=jp
-#EXTINF:-1 tvg-id="GSTV.jp" status="error",GSTV (720p)
-https://gemstv.wide-stream.net/gemstv01/smil:gemstv01.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GSTV.jp" status="online",GSTV (720p)
+https://gstv-tnz-gsmediastreaming.preview-jpea.channel.media.azure.net/dfd06b62-e9d1-4a7f-bcbb-89d2ecbc82ee/preview.ism/manifest(format=mpd-time-csf,audio-only=false)
```
Comment (freearhey) [inline streams/jp.m3u:52]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/jp.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/jp.m3u:52]: manifest(format=m3u8-aapl,audio-only=false)_snapshot_00 17 442

Works badly indeed, only seems to load correctly on browser

**Review by freearhey — APPROVED** [review]

**Pull request merged**